flixel/system/FlxPreloader.hx

```diff
@@ -48,20 +48,18 @@ class FlxPreloader extends FlxBasePreloader
 	 */
 	override private function create():Void
 	{
+		#if !js
 		_buffer = new Sprite();
 		_buffer.scaleX = _buffer.scaleY = 2;
 		addChild(_buffer);
 		_width = Std.int(Lib.current.stage.stageWidth / _buffer.scaleX);
 		_height = Std.int(Lib.current.stage.stageHeight / _buffer.scaleY);
 		_buffer.addChild(new Bitmap(new BitmapData(_width, _height, false, 0x00345e)));
-		
-		var logoLight = createBitmap(GraphicLogoLight, function(logoLight:Bitmap)
```

Comment:
Perhaps not all recent changes have been merged with this? Quite an odd regression.